Brokerages/InteractiveBrokers/Client/SecurityType.cs

```diff
@@ -75,6 +75,11 @@ public static class SecurityType
         /// </summary>
         public const string Bill = "BILL";
 
+        /// <summary>
+        /// Contract For Difference
+        /// </summary>
+        public const string ContractForDifference = "CFD";
```

Comment:
This doesn't seem used anywhere?

Reply:
Yes, I just added it for completeness -- also maybe we'll add CFD support for IB in the future?